lib/resolve.js

```diff
@@ -101,35 +102,34 @@ function select($, selector) {
  * @return {String}
  */
 
-function attribute($el, attr) {
-  switch(attr) {
+function attribute ($el, attr) {
+  switch (attr) {
     case 'html':
-      return $el.html();
+      return $el.html()
     case 'text':
-      return $el.text();
+      return $el.text()
     default:
-      return $el.attr(attr);
+      return $el.attr(attr)
   }
 }
 
-
-// var cheerio = require('cheerio');
-// var $ = cheerio.load('<ul title="some title"><li>a</li><li>b</li></ul><ul><li>c</li><li>d</li></ul>');
+// var cheerio = require('cheerio')
```

Comment:
not me, a robot for me!

Read basic checklist related with [standard](https://github.com/feross/standard#rules) style.